libs/community/langchain_community/embeddings/baichuan.py

```diff
@@ -79,15 +94,14 @@ def _embed(self, texts: List[str]) -> Optional[List[List[float]]]:
                 return [result.get("embedding", []) for result in sorted_embeddings]
             else:
                 # Log error or handle unsuccessful response appropriately
-                print(  # noqa: T201
+                # Handle 100 <= status_code < 400, not include 200
+                raise RequestException(
                     f"Error: Received status code {response.status_code} from "
-                    "embedding API"
+                    "`BaichuanEmbedding` API"
                 )
-                return None
-        except Exception as e:
+        except Exception:
```

Comment:
can remove the try and except now since it doesn't do anything?

Reply:
done